src/pages/AssetPage/index.tsx

```diff
@@ -121,6 +125,27 @@ export class AssetPage extends React.Component<AssetPageProps> {
     </div>
   );
 
+  private renderDepositMenuItem = (
+    route: string,
+    iconUrl: string,
+    label: string
+  ) =>
+    this.profileStore.isKycPassed ? (
```

Comment:
It could be two different render methods

Reply:
For me it's pretty small and straightforward function